### PR TITLE
Crud security - Functional tests

### PR DESCRIPTION
**Pull request by stafyniaksacha**

### CODE REVIEW

**Comment by AnthonySendra** [comment]
+1

File: core/views/role/full.hbs [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
     </div>
 
     <div class="col-sm-10 col-sm-offset-1" ng-if="!notFoundError">
 
         <form>
             <div class="form-group">
-                <label for="name">Role name</label>
-                <input type="text" class="form-control" ng-model="role.name" id="name" ng-disabled="isEdit">
+                <label for="id">Role name</label>
+                <input type="text" class="form-control" ng-model="role.id" id="id" ng-disabled="isEdit">
```
Comment (xbill82) [inline core/views/role/full.hbs]: `id="id"`? We want to avoid this, don't we?

File: core/views/user/full.hbs [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
     </div>
 
     <div class="col-sm-10 col-sm-offset-1" ng-if="!notFoundError">
 
         <form>
             <div class="form-group">
-                <label for="name">User id</label>
-                <input type="text" class="form-control" ng-model="user.id" id="name" ng-disabled="isEdit">
+                <label for="id">User id</label>
+                <input type="text" class="form-control" ng-model="user.id" id="id" ng-disabled="isEdit">
```
Comment (xbill82) [inline core/views/user/full.hbs]: Same as above.

File: public/javascripts/common/roleApi.service.js [hunk 3]
```diff
@@ -49,7 +49,9 @@ angular.module('kuzzle.roleApi', ['ui-notification', 'kuzzle.kuzzleSdk'])
             messageSuccess = 'Role updated !';
           }
 
-          kuzzleSdk.security.createRole(role.name, role.content, {replaceIfExist: true}, function (error, role) {
+          console.log('api:role:update', role);
```
Comment (xbill82) [inline public/javascripts/common/roleApi.service.js]: Clean this up?

**Comment by j33f** [comment]
+1 (and +1 for @xbill82 suggestions)

**Comment by xbill82** [comment]
+1

**Pull request merged**